src/pj_init.c

```diff
@@ -559,7 +559,16 @@ pj_init_ctx(projCtx ctx, int argc, char **argv) {
     /* longitude center for wrapping */
     PIN->is_long_wrap_set = pj_param(ctx, start, "tlon_wrap").i;
     if (PIN->is_long_wrap_set)
+    {
         PIN->long_wrap_center = pj_param(ctx, start, "rlon_wrap").f;
+        /* Don't accept excessive values otherwise we might perform badly */
+        /* when correcting longitudes around it */
+        if( !(fabs(PIN->long_wrap_center) < 10 * M_TWOPI) )
```

Comment:
Perhaps a formulaic comment "Backwards test to allow NaNs to satisfy the condition" is called for.